test/integration/testManualConnection.js

```diff
@@ -330,4 +331,34 @@ if (process.env.RUN_MANUAL_TESTS_ONLY === 'true') {
       assert.ok(sumWithoutKeepAlive * 0.66 > sumWithKeepAlive, 'With keep alive the queries should work faster');
     });
   });
+
+
+  // Before run below tests you should prepare files connections.toml and token
+  describe('Connection file configuration test', function () {
+    afterEach(async function () {
+      delete process.env.SNOWFLAKE_HOME;
+      delete process.env.SNOWFLAKE_DEFAULT_CONNECTION_NAME;
+    });
+
+    it('test simple connection', async function () {
+      const configuration = await laodConnectionConfiguration();
+      await verifyConnectionWorks(configuration);
+    });
+    it('test connection with token', async function () {
+      process.env.SNOWFLAKE_DEFAULT_CONNECTION_NAME = 'aws-oauth';
+      const configuration = await laodConnectionConfiguration();
+      await verifyConnectionWorks(configuration);
+    });
+    it('test connection with token from file', async function () {
+      process.env.SNOWFLAKE_DEFAULT_CONNECTION_NAME = 'aws-oauth-file';
+      const configuration = await laodConnectionConfiguration();
+      await verifyConnectionWorks(configuration);
+    });
+
+    async function verifyConnectionWorks(configuration) {
+      const connection = snowflake.createConnection(configuration);
+      await testUtil.connectAsync(connection);
+      assert.ok(connection.isUp(), 'not active');
```

Comment:
let's add `select 1` and teminate connection

Reply:
Fixed